ipatests/test_integration/test_acme.py

```diff
@@ -871,6 +871,9 @@ def test_prune_cert_cron(self, issue_and_expire_acme_cert):
                 ),
             ]
         ).stdout_text.strip()
+        # cases where cron_minute + 5 > 60
+        if int(cron_minute) > 59:
+            cron_minute = int(cron_minute) % 60
```

Comment:
Since you are using python to get the minutes, you can also leverage timedelta object:
```
from datetime import datetime; from datetime import timedelta; print(int((datetime.now() + timedelta(minutes=5)).strftime('%M')))
```
The methods already take care of the minutes overflow.